build/yamls/antrea-eks.yml

```diff
@@ -324,6 +324,9 @@ metadata:
 apiVersion: v1
 data:
   antrea-agent.conf: |
+    # FeatureGates is a map of feature names to bools that enable or disable alpha/experimental features.
```

Comment:
Maybe remove "alpha", as it is more for experimental features?

Reply:
Done